src/test/unit/util.hpp

```diff
@@ -59,6 +59,36 @@ void match_csv_columns(const Eigen::MatrixXd& samples,
   }
 }
 
+/**
+ * Tests for  exact elementwise equality of the input matrices
+ * with the EXPECT_EQ macro from GTest.
+ *
+ * @param A first input matrix to compare
+ * @param B second input matrix to compare
+ */
+#ifndef EXPECT_MATRIX_EQ
```

Comment:
We tend to keep the tests for stan math separate so I'd rather define it here. idt anywhere else in the testing for the stan repo do we pull out anything from the math testing repo. This is just so that in the future if we ever merge the two libraries the macro is not defined twice